app/services/bulk_tagging/fetch_tagged_content.rb

```diff
@@ -12,6 +12,30 @@ def initialize(content_id, document_type)
     end
 
     def call
+      if taxon_document_type?
+        request_tagged_content_to_taxon
+      else
+        request_tagged_content_items
+      end
+    end
+
+  private
+
+    def taxon_document_type?
+      document_type == "taxon"
+    end
+
+    def request_tagged_content_to_taxon
```

Comment:
Thanks @tijmenb, I've changed it, you can see it in this commit: https://github.com/alphagov/content-tagger/pull/172/commits/8c87e8057926ed5c0485d5519729a47b4c9b0709